cmd/proxy/actions/filter.go

```diff
@@ -30,6 +38,18 @@ func newFilterMiddleware(mf *module.Filter) buffalo.MiddlewareFunc {
 			// i.e. list requests path is like /{module:.+}/@v/list with no version parameter
 			version, _ := paths.GetVersion(c)
 
+			if validatorHook, ok := env.ValidatorHook(); ok && version != "" {
```

Comment:
can we separate this into its own middleware? then we can have FCB to enable/disable this